packages/preview/tada/0.1.0/lib.typ

```diff
@@ -0,0 +1,6 @@
+#import "src/ops.typ": *
+#import "src/tabledata.typ": *
+#import "src/display.typ": *
+#import "src/ops.typ"
+#import "src/tabledata.typ"
+#import "src/display.typ"
```

Comment:
are these intentionally star imports (which could potentially leak a lot of internals)? should it maybe be a more selected list of things?